src/screens/Forms/FormsScreen.tsx

```diff
@@ -1,13 +1,93 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import GeneralQuestionManager from 'components/GeneralQuestionManager/GeneralQuestionManager';
 import { ScrollPageContainer } from 'screens/styles';
+import DacaRenewalQuestionManager from 'components/DacaRenewalQuestionManager/DacaRenewalQuestionManager';
+import { Client, Dictionary } from 'types/types';
+import { getCurrentClient } from 'database/auth';
+import FinalIntakeScreen from 'components/FinalIntakeScreen/FinalIntakeScreen';
+
+// TODO: integrate user auth, retention of answers.
 
 const FormsScreen = () => {
-  return (
-    <ScrollPageContainer>
-      <GeneralQuestionManager />
-    </ScrollPageContainer>
-  );
+  const [showAdditionalScreen, setShowAdditionalScreen] = useState(false);
+  const [showFinalScreen, setShowFinalScreen] = useState(false);
+  const [showGeneralScreen, setShowGeneralScreen] = useState(true);
+  const [additionalScreenType, setAdditionalScreenType] = useState('');
+  const [generalScreenNumber, setGeneralScreenNumber] = useState(0);
+  const [existingAnswers, setExistingAnswers] = useState(new Map());
+
+  const loadClient = async (): Promise<void> => {
+    const client: Client | undefined = await getCurrentClient();
+    if (client) {
+      setExistingAnswers(client.answers);
+    }
+  };
+
+  useEffect(() => {
+    loadClient();
+  }, []);
+
+  const setAdditionalScreen = (visitReason: string): void => {
+    loadClient();
+    setAdditionalScreenType(visitReason);
+    setShowAdditionalScreen(true);
+    setShowGeneralScreen(false);
+    setShowFinalScreen(false);
+    setGeneralScreenNumber(5);
```

Comment:
Nice way of jumping to the last general screen!

Reply:
[BUG] I notice that when I go back from DACA screens to the last general question page, the dropdown + date answers do not save, causing client to repick their case type again. This might be due to my refactoring of dropdown/calendar tho, so don't worry too much about it for this sprint